project/activities/views.py

```diff
@@ -7,6 +7,73 @@
 
 from .forms import ActivityModelForm
 from .models import Activity
+from .serializers import ActivitySerializer
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+@api_view(['GET', 'POST'])
+def activity_list(request):
```

Comment:
For cleaner code, let's use class-based API views.

https://www.django-rest-framework.org/tutorial/3-class-based-views/